src/types.js

```diff
@@ -3015,7 +3015,7 @@ function isJsonBuffer(obj) {
  *
  * @param path {Array} Passed from hook, but unused (because empty where this
  * function is used, since we aren't keeping track of it for effiency).
- * @param val {...} The object to reject.
+ * @param val {any} The object to reject.
```

Comment:
This fails otherwise with:

```
src/types.js:3018:16 - error TS1014: A rest parameter must be last in a parameter list.

3018  * @param val {...} The object to reject.
                    ~~~


Found 1 error in src/types.js:3018
```